backend/src/ml_space_lambda/app_configuration/lambda_functions.py

```diff
@@ -33,6 +46,13 @@ def update_configuration(event, context):
 
     new_configuration = SettingsModel.from_dict(request["configuration"])
 
+    previous_app_configurations = app_configuration_dao.get(configScope=configScope, num_versions=1)
```

Comment:
Posting here for posterity; Dustin and I discussed doing that same thing as the enabling/disabling services feature is going to do, which is we're not going to bother comparing this updating config against prior configs, but rather we'll simply generate an IAM policy based on whatever the latest config is and update the IAM policy using that.

Reply:
Yeah that doesn't need to exist anymore now that we're not comparing.